homeassistant/components/yeelight/light.py

```diff
@@ -175,6 +176,10 @@
     vol.Required(ATTR_MODE): vol.In([mode.name.lower() for mode in PowerMode])
 }
 
+SERVICE_SCHEMA_SET_MUSIC_MODE = {
+    vol.Required(CONF_MODE_MUSIC): cv.boolean,
```

Comment:
We only use `CONF_*` constants for config items.
```suggestion
    vol.Required(ATTR_MODE_MUSIC): cv.boolean,
```

Reply:
Done